### PR TITLE
 Fixes broken link of contributors cheatsheet

### LINKED ISSUE #371: Broken link of `contributor cheatsheet` Need to fix
*This is the bug report*
This page  [CONTRIBUTING.md](https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/CONTRIBUTING.md) have one broken link of `Contributor Cheat Sheet` which needs to fix

/kind bug
/assign

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes #371

### CODE REVIEW

**Review by ggriffiths — APPROVED** [review]
/lgtm

**Comment by ggriffiths** [comment]
/assign @xing-yang 

for approval

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
 - [Kubernetes Contributor Guide](http://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet.md) - Common resources for existing developers
+- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet) - Common resources for existing developers
```
Comment (amolmote) [inline CONTRIBUTING.md:17]:
```suggestion
- [Contributor Cheat Sheet](https://k8s.dev/cheatsheet) - Common resources for existing developers
```

**Comment by xing-yang** [comment]
Can you please address the comment?

**Pull request closed without merging**